### PR TITLE
Move bucket-blocker into general tool

### PR DESCRIPTION
**Pull request by NovemberTang**
## What does this change?

Previously, fixing issues meant installing a tool per issue. We hope to simplify this process by having one tool, that teams can use via subcommand. This PR starts that process, creating an  fsbp-fix module, and moving bucket-blocker into it. Eventually, all the code will move into here and we can move away from the monorepo model

## How to test

CI passes

## How can we measure success?

The toolset becomes easier to maintain, and distribute.

### CODE REVIEW

File: go.work [hunk 1]
```diff
@@ -1,7 +1,7 @@
 go 1.22.1
 
-use ./bucket-blocker
-
 use ./ingress-inquisition
 
 use ./common
+
+use ./fsbp-fix
```
Comment (NovemberTang) [inline go.work:7]: after all the other projects have moved into this one, we can move it to the root

File: fsbp-fix/bucket-utils/awsutils.go [hunk 2]
```diff
@@ -17,7 +17,6 @@ import (
 func findFailingBuckets(ctx context.Context, securityHubClient *securityhub.Client, bucketCount int32) ([]string, error) {
 	controlId := "S3.8"
 
-	fmt.Println("Retrieving Security Hub control failures for S3.8")
```
Comment (NovemberTang) [inline fsbp-fix/bucket-utils/awsutils.go:20]: Deleting as this log line already shows up in ReturnFindings()

File: fsbp-fix/main.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+		execute := fixS3_8.Bool("execute", false, "Execute the block operation")
+		profile := fixS3_8.String("profile", "", "AWS profile to use")
+		region := fixS3_8.String("region", "", "The region of the bucket")
+		bucketCount := fixS3_8.Int("max", 100, "The maximum number of buckets to attempt to process")
+		exclusions := fixS3_8.String("exclusions", "", "Comma-separated list of buckets to skip")
+
+		fixS3_8.Parse(os.Args[2:])
+
+		if *profile == "" {
```
Comment (NovemberTang) [inline fsbp-fix/main.go:40]: In a follow-up PR, I'll introduce a new function that does all the nil value checks in one line. For now, I'm just lifting and shifting

**Review by tjsilver — COMMENTED** [review]
Is the plan to move everything before we send out the FSBP comms? It could be confusing to people who have been made aware of bucket blocker and ingress-inquisition as separate tools

**Review by tjsilver — APPROVED** [review]
Looks good, subject to comms being adjusted 👍

**Pull request merged**